test/bin/ci_phase_iso_build.sh

```diff
@@ -115,6 +116,10 @@ $(dry_run) bash -x ./scripts/image-builder/configure.sh
 
 cd "${ROOTDIR}/test/"
 
+# Source common.sh only after all dependencies are installed.
```

Comment:
Do we need this change in the main branch, too?

Reply:
Yes, though it accidentally works as the fallback is `main`

Reply:
OK, please make sure there's a jira ticket for it.

Reply:
We need that on `release-4.15` as well